lib/parsers.js

```diff
@@ -15,7 +15,8 @@ module.exports.parseText = function parseText(message, kwargs) {
 
 module.exports.parseError = function parseError(err, kwargs, cb) {
   utils.parseStack(err, function(frames) {
-    var name = err.name + '';
+    var name =
+      ({}.hasOwnProperty.call(err, 'name') ? err.name : err.constructor.name) + '';
```

Comment:
Please use `Object.prototype.hasOwnProperty.call()` since you don't create a new object first, furthermore I think the constructor name should be leading

Reply:
Thanks for pointing that out. However, it's such a micro-optimization, which won't make any difference ever (this object will be collected by GC anyway), that I'll just patch it while working on something else and won't create a separate PR for it. We also use `{}.` notation in other places, and I prefer code consistency over small performance gains :)